scripts/datasets/utils.py

```diff
@@ -0,0 +1,161 @@
+"""Utility functions for transforming datasets into the DialogueKit format."""
```

Comment:
Nit: "into the DialogueKit format" => "into DialogueKit format"

Reply:
Done.